test/predicates.jl

```diff
@@ -130,6 +130,14 @@
   end
 
   @testset "intersects" begin
+    function assertintersecsundertransforms(g1, gt, gf)
```

Comment:
We can remove this function and copy/paste the for loop directly in the test, after the definition of the two geometries being tested.